internal/server/models/settings.go

```diff
@@ -5,6 +5,4 @@ type Settings struct {
 
 	PrivateJWK EncryptedAtRestBytes
 	PublicJWK  []byte
-
-	SignupEnabled bool
```

Comment:
How does GORM deal with deleted fields? Does this need any kind of migration?

Reply:
I expected it to drop the column but it looks like it doesn't so will add a migration instead